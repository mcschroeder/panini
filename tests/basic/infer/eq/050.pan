eq : (a:ℤ) → (b:ℤ) → { c:𝔹 | c = true ⟺ a = b }

f50 = \x:int. eq x 1

f51 = \x:int. eq 1 x

