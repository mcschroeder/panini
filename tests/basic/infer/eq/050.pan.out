f50 : {x:ℤ | true} → {c:𝔹 | c = true ⇔ x = 1}
f51 : {x:ℤ | true} → {c:𝔹 | c = true ⇔ 1 = x}